src/01-functions.js

```diff
@@ -168,7 +191,10 @@ function joinStringsWithSpaces() {}
  *  getFirstAndLastCharacter("upwards");
  *  //> "us"
  */
-function getFirstAndLastCharacter() {}
+function getFirstAndLastCharacter(a) {
+
+  return `${a[0]}${a[a.length - 1]}`;
```

Comment:
Using template literals when most or all of the string is evaluated JS can be tough to read. You might try `a[0] + a[a.length - 1]` for clearer code.